virttest/qemu_monitor.py

```diff
@@ -1725,6 +1725,13 @@ def cmd(self, cmd, args=None, timeout=CMD_TIMEOUT, debug=True, fd=None):
         finally:
             self._lock.release()
 
+    @staticmethod
+    def _build_args(**kargs):
+        """
+        Build args used in cmd.
+        """
+        return {k: v for k, v in kargs.items() if v is not None}
```

Comment:
There is another thing should be done within this function: to convert the arguments' name by replacing `_` to `-`, where the former is accepted by python while the latter is accepted by QMP.